src/formpack/version.py

```diff
@@ -177,6 +179,10 @@ def __init__(self, form_pack, schema):
                 parent_section.children.append(section)
                 continue
 
+            if 'disabled' in data_definition:
```

Comment:
is it illegal for groups to be disabled?

Reply:
good catch

Reply:
fixed in 1234c8